<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Edit_device</class>
 <widget class="QDialog" name="Edit_device">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="OK_btn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background: rgb(45, 81, 135);
border-radius: 12px;</string>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>60</y>
     <width>311</width>
     <height>230</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="4" column="0">
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>Size of display:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="usb_lineEdit_2"/>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="size_lineEdit_2"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>Maximum refreah rate:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string>field of view:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QCheckBox" name="with_controller_2">
      <property name="text">
       <string>controller</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="display_lineEdit_2"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="max_refreah_lineEdit_2"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_19">
      <property name="text">
       <string>Display:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="field_v_lineEdit_2"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_20">
      <property name="text">
       <string>USB type:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>351</width>
     <height>232</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="5" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Resolution:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="quantity_lineEdit_2"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="brand_lineEdit_2"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_21">
      <property name="text">
       <string>Model:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_22">
      <property name="text">
       <string>Description:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_23">
      <property name="text">
       <string>Quantity:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="des_lineEdit_2"/>
    </item>
    <item row="5" column="2">
     <widget class="QLabel" name="label_24">
      <property name="text">
       <string>X</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="price_lineEdit_2"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_25">
      <property name="text">
       <string>Brand:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QLineEdit" name="resolution_lineEdit_3"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_26">
      <property name="text">
       <string>Price:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="model_lineEdit_2"/>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="resolution_lineEdit_4"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 22pt &quot;Tw Cen MT Condensed Extra Bold&quot;;
color: rgb(0, 85, 127);</string>
   </property>
   <property name="text">
    <string>EDIT (VR)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
